pkgs/by-name/ku/kubetui/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  rustPlatform,
+  lib,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kubetui";
+  version = "1.5.3";
+
+  src = fetchFromGitHub {
+    owner = "sarub0b0";
+    repo = "kubetui";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-0K0h/MaQClJDqgF0qQO2INb+hpzxfSikAti+751MX/8=";
+  };
+
+  cargoHash = "sha256-kfAErXqjdLn2jaCtr+eI0+0v4TcE8Hpx0DDECnuYp5w=";
+
+  doCheck = false;
```

Comment:
Rather than completely disable testing, wouldn't it be better to just disable the failing test case?

```
  checkFlags = [
    "--skip=workers::kube::store::tests::kubeconfigからstateを生成"
  ];
```

Reply:
still has `doCheck = false`